contracts/bico-token/bico/PolygonBicoToken.sol

```diff
@@ -0,0 +1,44 @@
+// File: contracts/child/ChildToken/UpgradeableChildERC20/UChildERC20.sol
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.4;
+
+import "./BicoTokenImplementation.sol";
+
+contract PolygonBicoToken is Initializable, BicoTokenImplementation {
+    bytes32 public constant DEPOSITOR_ROLE = keccak256("DEPOSITOR_ROLE");
+
+    /**
+     * @notice Initialize the contract after it has been proxified
+     * @dev meant to be called once immediately after deployment
+     */
+    function polygonBico_init( address beneficiary, address trustedForwarder, address governor, address accessControlAdmin, address pauser, address minter, address childChainManager ) public initializer {
+        initialize(beneficiary, trustedForwarder, governor, accessControlAdmin, pauser, minter);
```

Comment:
Need to change in BicoTokenImplementation also to not mint any tokens in initialise function if beneficiary is address(0)

Reply:
Also write test case for this.